shared/AppInsightsCore/Tests/Perf/src/CorePerfCheck.Tests.ts

```diff
@@ -459,7 +492,7 @@ export class CorePerfCheckTests extends AITestClass {
                 let checks = 0;
                 return this._runPerfTestAsync("baseTestObject", () => {
                     JSON.stringify(baseTestObject);
-                }, 50, iterations, 0.005).then((baseDuration) => {
+                }, 60, iterations, 0.005).then((baseDuration) => {
```

Comment:
Also increasing the minimum time for the same reason -- build servers are virtualized slow beasts